src/App.tsx

```diff
@@ -17,18 +53,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                filterType={filterType}
+                setFilterType={setFilterType}
+                query={query}
+                setQuery={setQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!isLoading
```

Comment:
```suggestion
              {isLoading
```
навіщо перетворювати булеве в противоположне значення?
зміни порядок опцій в тернарному операторі тай все))